README.md

```diff
@@ -86,8 +86,8 @@ If you are familiar with Web IDL, you may also want to check whether the upstrea
 - `addedTypes.jsonc`: types that should exist but are missing from the spec data.
 - `overridingTypes.jsonc`: types that are defined in the spec but have TypeScript-friendly modifications in the json files.
 - `removedTypes.jsonc`: types that are defined in the spec but should be removed.
-- `comments.json`: comment strings to be embedded in the generated .js files.
-- `deprecatedMessage.json`: the reason why one type is deprecated. The reason why it is a separate file rather than merge in comment.json is mdn/apiDescriptions.json would also possibly be deprecated.
+- `comments.json`: comment strings to be embedded in the generated .d.ts files.
```

Comment:
(I think this is mostly deprecated after your MDN patch. We should consider removing it)